week06/6.2.timer.py

```diff
@@ -0,0 +1,51 @@
+#------------------------
+# ריצת 2000 - 6.2
+#------------------------
+
+#===========================================================================
+# timer function:
+# gets a function and unlimited number of parameters for that function.
+# returns the time it took to execute the function with those parameters.
+#===========================================================================
+import timeit
+
+def timer(func, *args):
+    start = timeit.default_timer()
+    func(*args)
+    end = timeit.default_timer()
+    # return in seconds
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/

Reply:
Fixed now